hetmech/degree_group.py

```diff
@@ -72,13 +72,11 @@ def compute_summary_metrics(df):
 
 def dwpc_to_degrees(graph, metapath, damping=0.5):
```

Comment:
What do the changes in this fxn do?

Reply:
Two things:
1. Fix the error whereby the `path_counts` column shows `dwpc` values instead.
2. Return only rows having `dwpc` > 0. This was necessary because `G---G` metapaths were exceeding the memory on my computer when bringing these into pandas dataframes.

Reply:
Can we add a docstring:

> Yield a description of each cell in a DWPC matrix adding source and target node degree info as well as the corresponding path count.

Reply:
I think we want the make skipping zero-PC cells an option but not necessarily the default or only option.